CONTRIBUTING.md

```diff
@@ -24,4 +24,44 @@ We use the
 and review all changes we merge to master.
 
 We appreciate any contributions, so please use the [Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow)
-and send us `Merge Requests`
+and send us `Merge Requests`. 
+
+## Developer's Certificate of Origin 1.1
+
+In order to pass the checks, commit with signoff option
+
+```shell
+git commit --signoff
+```
+
+When signing-off a patch for this project like this
+
+    Signed-off-by: Random J Developer <random@developer.example.org>
+
+using your real name (no pseudonyms or anonymous contributions), you declare the
+following:
+
+    By making a contribution to this project, I certify that:
+
+        (a) The contribution was created in whole or in part by me and I
+            have the right to submit it under the open source license
+            indicated in the file; or
+
+        (b) The contribution is based upon previous work that, to the best
+            of my knowledge, is covered under an appropriate open source
+            license and I have the right under that license to submit that
+            work with modifications, whether created in whole or in part
+            by me, under the same open source license (unless I am
+            permitted to submit under a different license), as indicated
+            in the file; or
+
+        (c) The contribution was provided directly to me by some other
+            person who certified (a), (b) or (c) and I have not modified
+            it.
+
+        (d) I understand and agree that this project and the contribution
+            are public and that a record of the contribution (including all
+            personal information I submit with it, including my sign-off) is
+            maintained indefinitely and may be redistributed consistent with
+            this project or the open source license(s) involved.
```

Comment:
We do not have a template for that yet, also because some project were pushed toward CLA while other (like ours) are luckily fine with DCO. So these lines were basically taken from EFI Boot Guard, and that took them from Jailhouse once.